src/pages/PollingPage.tsx

```diff
@@ -10,15 +19,28 @@ import { Button, Card, Form, ListGroup, Spinner } from "react-bootstrap";
 import { BiExpand } from "react-icons/bi";
 import { AiOutlineBarChart } from "react-icons/ai";
 import Modal from "react-bootstrap/Modal";
-
+import { updateVote } from "../redux/VoteCountSlice";
 interface DeletePollType {
   id: number;
   index: number;
 }
+
+ChartJS.register(BarElement, CategoryScale, LinearScale, Tooltip, Legend);
 const PollingPage: React.FC = () => {
   const navigate = useNavigate();
   const polls = useSelector((state: RootState) => state.pollList);
-  const [votedItems, setVotedItems] = useState<string[]>([]);
+  const [singlePollInformationData, setSinglePollInformationData] = useState({
+    labels: [],
+    datasets: [
```

Comment:
if this a Poll information, why I am not seeing title and Options keys ? @Rishab-innovative